AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/IdentifyTextResultTransformers.swift

```diff
@@ -218,6 +238,14 @@ class IdentifyTextResultTransformers: IdentifyResultTransformers {
         return cell
     }
 
+    // https://docs.aws.amazon.com/textract/latest/dg/how-it-works-kvp.html
+    /**
+    * Converts a given Amazon Textract block into Amplify-compatible
+    * key-value pair feature. Returns null if not a valid table.
+    * @param block Textract text block
+    * @param blockMap map of Textract blocks by their IDs
+    * @return Amplify KeyValue instance
+    */
```

Comment:
Lets remove this comment.  We typically have these types of comments for public facing functions that don't change very often.  This seems like a utility function, that is only used local to this file, right?